docs/rules/block-lang.md

```diff
@@ -21,7 +21,7 @@ This rule enforces all svelte components to use the same set of languages for th
 ```svelte
 <!-- ✓ GOOD -->
 <script lang="ts">
-/* eslint svelte/block-lang: {"error", { "script": "ts" }} */
+  /* eslint svelte/block-lang: {"error", { "script": "ts" }} */
```

Comment:
I use `prettier` for formatting instead of `remark`, few formatting occurs but I think these are right formatting.